GameServer.cpp

```diff
@@ -618,6 +612,23 @@ int GameServer::ReceiveFrom(Packet &packet, sockaddr_in &from)
 	return numBytes;

```

Comment:
This is return a size_t as an int. In practice, I'm not worried since I don't see how we could exceed int in size for a packet. Changing it would force an update to the error enum and restructuring another function. Maybe this is work for another issue?